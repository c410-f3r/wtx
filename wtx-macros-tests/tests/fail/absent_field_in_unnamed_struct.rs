#[wtx_macros::pkg(data_format(json), id(FooId))]
mod pkg {
  #[pkg::aux]
  impl Foo {}

  #[pkg::req_data]
  struct Req(
    i32
  );

  #[pkg::res_data]
  struct Res;
}

fn main() {
}
