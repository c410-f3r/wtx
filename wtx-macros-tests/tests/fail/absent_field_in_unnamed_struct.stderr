error: Unnamed structures must have a `#[pkg::field]` attribute on each field.
 --> tests/fail/absent_field_in_unnamed_struct.rs:7:10
  |
7 |   struct Req(
  |          ^^^

error: aborting due to 1 previous error

